example/matlab/AutogenerationExample_grt_rtw/AutogenerationExample.cpp

```diff
@@ -24,382 +24,429 @@
 /* Model step function */
```

Comment:
The big change of style is due to a different clang-format? Or a change in simulink coder output?

Reply:
I just realized that in the very beginning I committed by mistake the `.h` without any style, and the `.cpp` styled. Here I forgot to apply the clang-format to the `.cpp`. I did not realize it :sweat_smile:

Let's keep the style consistent, I will format everything and update the PR.